deployments/azure-webapps-node.yml

```diff
@@ -26,7 +26,7 @@ on:
 env:
   AZURE_WEBAPP_NAME: your-app-name    # set this to your application's name
   AZURE_WEBAPP_PACKAGE_PATH: '.'      # set this to the path to your web app project, defaults to the repository root
-  NODE_VERSION: '14.x'                # set this to the node version to use
+  NODE_VERSION: '16.x'                # set this to the node version to use
```

Comment:
@JasonFreeberg please sign off on this change

Reply:
Leave a reply

Reply:
> @JasonFreeberg please sign off on this change

How can l do it?